app/src/org/odk/collect/android/utilities/NepaliDateUtilities.java

```diff
@@ -0,0 +1,347 @@
+package org.odk.collect.android.utilities;
+
+import static org.odk.collect.android.utilities.UniversalDate.MILLIS_IN_DAY;
+
+import java.util.Date;
+
+import android.util.SparseArray;
+
+/**
+ * Date manipulation utility functions for the Nepali calendar system.
+ * 
+ * @author Richard Lu
+ */
+public class NepaliDateUtilities {
+    
+    private static final SparseArray<int[]> NEPALI_YEAR_MONTHS = new SparseArray<int[]>();
+    
+    public static final int MIN_YEAR = 1970;
```

Comment:
1970 BS = 1914 AD (our years), so this lets us register people up to 101 years of age.